go/exocom/features/translating-messages.feature

```diff
@@ -23,6 +23,16 @@ Feature: Translating messages
         "role": "tweets",
         "receives": ["text-snippets.create"],
         "sends": ["text-snippets.created"],
+        "messageTranslation:" [
+          {
+            "public":"tweets.create",
+            "internal":"text-snippets.create"
```

Comment:
@kevgo we thought it made a little more sense to use public and internal as the keys instead of dome / internal? What do you think? @mcharawi if we stick with this please update the documentation.